src/App.tsx

```diff
@@ -1,196 +1,10 @@
-        'soldier',
-        'knight',
-        'cavalier',
-        'archer',
-        'sniper',
-        'mage',
-        'ordnance',
-        'monarch',
-        'lord',
-        'demi-human',
-        'outsider',
-        'giant',
-        'horse',
-        'dog',
-        'wolf',
-        'rat',
-        'beast',
-        'bird',
-        'raptor',
-        'snake',
-        'crab',
-        'prawn',
-        'octopus',
-        'bug',
-        'scarab',
-        'slug',
-        'wraith',
-        'skeleton',
-        'monstrosity',
-        'ill-omened creature',
-    ],
-    'People': [
-        'Tarnished',
-        'warrior',
-        'swordfighter',
-        'knight',
-        'samurai',
-        'sorcerer',
-        'cleric',
-        'sage',
-        'merchant',
-        'teacher',
-        'master',
-        'friend',
-        'lover',
-        'old dear',
-        'old codger',
-        'angel',
-        'fat coinpurse',
-        'pauper',
-        'good sort',
-        'wicked sort',
-        'plump sort',
-        'skinny sort',
-        'lovable sort',
-        'pathetic sort',
-        'strange sort',
-        'numble sort',
-        'laggardly sort',
-        'invisible sort',
-        'unfathomable sort',
-        'giant sort',
-        'sinner',
-        'thief',
-        'liar',
-        'dastard',
-        'traitor',
-        'pair',
-        'trio',
-        'noble',
-        'aristocrat',
-        'hero',
-        'champion',
-        'monarch',
-        'lord',
-        'god',
-    ],
-    'Things': [
-        'item',
-        'necessary item',
-        'precious item',
-        'something',
-        'something incredible',
-        'treasure chest',
-        'corpse',
-        'coffin',
-        'trap',
-        'armament',
-        'shield',
-        'bow',
-        'projectile weapon',
-        'armor',
-        'talisman',
-        'skill',
-        'sorcery',
-        'incantation',
-        'mao',
-        'material',
-        'flower',
-        'grass',
-        'tree',
-        'fruit',
-        'seed',
-        'mushroom',
-        'tear',
-        'crystal',
-        'butterfly',
-        'bug',
-        'dung',
-        'grace',
-        'door',
-        'key',
-        'ladder',
-        'lever',
-        'lift',
-        'spiritspring',
-        'sending gate',
-        'stone astrolabe',
-        'Birdseye Telescope',
-        'message',
-        'bloodstain',
-        'Erdtree',
-        'Elden Ring',
-    ],
-    'Battle Tactics': [],
-    'Actions': [],
-    'Situations': [],
-    'Places': [],
-    'Directions': [],
-    'Body Parts': [],
-    'Affinities': [],
-    'Concepts': [],
-    'Phrases': []
-} as Record<Word, string[]>
-
-type Word =
-    'Enemies' |
-    'People' |
-    'Things' |
-    'Battle Tactics' |
-    'Actions' |
-    'Situations' |
-    'Places' |
-    'Directions' |
-    'Body Parts' |
-    'Affinities' |
-    'Concepts' |
-    'Phrases'
+const {dictionary, templates} = data
```

Comment:
Currently, destructuring default imports (e.g. `import {dictionary, templates} from './data.json'`) is unsupported (though the error did imply support would be forthcoming)